Easy/3. COVID-19 Food Supply App/solutions/README.md

```diff
@@ -1 +1,92 @@
-# Solutions
+
```

Comment:
Please revert all the changes made to this file. This PR should contain only the changes to your folder, i.e., `Vanshikaa00_Vanshika` folder.

Reply:
Hi @irshadjsr21 i followed the above steps you mentioned to update my branch foodSupplyEasy and also made the changes you requested in README.md file of solutions folder but when i try to push the changes made in  foodSupplyEasy branch (commit: update 2), i am receiving errors and if i try the hints too, i still get an error . I tried to search for a solution on internet but that did'nt worked 😅
![Screenshot (1490)](https://user-images.githubusercontent.com/53425575/93779150-48f1c000-fc44-11ea-8854-458809177137.png)

Reply:
If your local repo has the right code, you can try `git push -f -u origin foodSupplyEasy`